cmd/kubeadm/app/cmd/cmd.go

```diff
@@ -51,14 +51,14 @@ func NewKubeadmCommand(f *cmdutil.Factory, in io.Reader, out, err io.Writer, env
 			    ┌──────────────────────────────────────────────────────────┐
 			    │  On the first machine                                    │
 			    ├──────────────────────────────────────────────────────────┤
-			    │ master# kubeadm init master                              │
+			    │ master# kubeadm init                                     │
 			    │ Your token is: <token>                                   │
```

Comment:
This isn't the format of the output we have ended-up with, what do you think we could do here? May be just remove the line?